scripts/jsign.mjs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 The Outline Authors
```

Comment:
Let's remove this file from the PR. I'd like to see how it will be used. I think we can have a script that does all the signing instead of having this generic wrapper.

Reply:
Sounds good.